assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java

```diff
@@ -10959,6 +10959,7 @@ public void startElement(String uri, String localName, String qName,
 							intString = attributes.getValue("numberoflogs");
 							try
 							{
+							if(!StringUtils.isBlank(intString)) {
```

Comment:
Hello. I didn't  thought necessary add  a comment on this commit because I thought that was clear in the code,  with the commentary that I make in jira, that parseInt of this String can throw a java.lang.NumberFormatException.
If It's helpful to add a comment in a commit, the next time I'll do.
On the subject of the indentation I have not done because I just wanted to add a line. If it were not so I would have to indent the entire block .